torch/csrc/quantized/quantized_backward.cpp

```diff
@@ -0,0 +1,76 @@
+#include <ATen/native/quantized/PackedParams.h>
+#include <torch/library.h>
+#include <torch/torch.h>
+
+namespace {
+using namespace torch::autograd;
+using namespace at;
+// This class is a custom gradient function that enables quantized tensor to pass input gradient
+// back to the previous layers
+// This function can be used when the user is adapting mixed precision for traninig after quantization
+// From torch layer, we have no access to linear_dynamic operator which needs to access via redispatching mechanism
+// TO-DO : currently we are supporting per tensor quantization only, will expand to per channel later on
+class PackedLinearWeightDynamicBackward :
+public Function<PackedLinearWeightDynamicBackward> {
+ public:
+  static torch::Tensor forward(
+    AutogradContext *ctx,
+    at::Tensor input,
```

Comment:
What's the difference between `at::Tensor` and `torch::Tensor`? And why use `at::` namespace if you already have `using namespace at;` on line 7
```suggestion
  static Tensor forward(
    AutogradContext *ctx,
    Tensor input,
```